lib/vrl/parser/src/ast.rs

```diff
@@ -758,18 +786,38 @@ impl fmt::Debug for Assignment {
         use Assignment::*;
 
         match self {
-            Single { target, expr } => write!(f, "{:?} = {:?}", target, expr),
-            Infallible { ok, err, expr } => write!(f, "Ok({:?}), Err({:?}) = {:?}", ok, err, expr),
+            Single { target, op, expr } => write!(f, "{:?} {} {:?}", target, op, expr),
+            Infallible { ok, err, op, expr } => {
+                write!(f, "Ok({:?}), Err({:?}) {} {:?}", ok, err, op, expr)
+            }
         }
     }
 }
 
-#[derive(PartialEq)]
+#[derive(Clone, PartialEq)]
 pub enum AssignmentTarget {
     Noop,
     Query(Query),
     Internal(Ident, Option<Path>),
-    External(Option<Path>),
+}
+
+impl AssignmentTarget {
+    pub fn into_expr(self, span: &Span) -> Expr {
+        match self {
+            AssignmentTarget::Noop => Expr::Literal(Node::new(span.clone(), Literal::Null)),
```

Comment:
Isn't `Span: Copy`? I wouldn't have expected the need to clone them.

Reply:
Oh, nice!